include/node.h

```diff
@@ -306,6 +319,10 @@ class Node : public NodeBase<Tdim> {
   //! A general velocity (non-Cartesian/inclined) constraint is specified at the
   //! node
   bool generic_boundary_constraints_{false};
+  //! Absorbing constraints
+  std::tuple<unsigned, double, double, double> absorbing_constraint_;
+  //! Absorbing traction
+  Eigen::Matrix<double, Tdim, 1> absorbing_traction_;
```

Comment:
I don't think we need to store this variable in node either

Reply:
`absorbing_traction_` removed